### DISCUSSION TITLE
Permission to read/write USB disks on Ubuntu

### DISCUSSION THREAD

**Discussion by martin-mmaus** [cstayyab/WALC#120] [Troubleshooting]
Hello. I need WALC permissions to read removable media to attach files from USB disks on my laptop. In Snap control panel I can't get that option. Is it possible to do from terminal? Any suggestions? Thanks in advance.

I get this message when I try to attach files from USB disks.
![image](https://user-images.githubusercontent.com/38982995/107832829-3b161600-6d70-11eb-94b3-735e2d0037b9.png)

**Comment by kotatsuman** [comment]
I also need it.
Normally, for Snap packages, you have to type `sudo snap connect nameoftheprogramwhichyouneedtoreadusbdevices:removable-media` to do the job, but it looks like such a module is not implemented in Walc yet.
